### PR TITLE
leopard microservice with grpc

### PR DESCRIPTION
**Pull request by mrrostam**

### CODE REVIEW

File: demo/go-grpc/readme.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+### Server
+
+Start the server first by running the following command from the demo folder:
+```console
+go run server.go \
+--access_key "${ACCESS_KEY}"
+```
+
+Replace `${ACCESS_KEY}` with yours obtained from Picovoice Console and `${AUDIO_PATH}` with a path to an audio file you
```
Comment (ksyeo1010) [inline demo/go-grpc/readme.md:42]: lets move ${AUDIO_PATH} stuff to client

File: demo/go-grpc/readme.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+wish to transcribe.
+
+### Client
+
+While the server is up, make a transcription request:
+
+```console
+```console
+go run client.go \
+--audio_file "${AUDIO_PATH}"
```
Comment (ksyeo1010) [inline demo/go-grpc/readme.md:52]: either put this as -input_audio or change client.go to use audio_file

**Review by ksyeo1010 — APPROVED** [review]

**Comment by ksyeo1010** [comment]
I think we should use 4 spaces instead of tabs

**Review by laves — CHANGES_REQUESTED** [review]
Code looks good!

**Pull request merged**